pages/Speakers.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts">
+import { useHead, useAPI, computed, getSpeakerSocialsLinks } from "#imports";
+import { NuxtImg, SectionTalkSpeakerPicture, Heading } from "#components";
+import { ASSOCIATION_NAME, EDITION } from "~/services/constants";
+
+import type { Ref } from "vue";
+import type { Talk, Speaker } from "@bdxio/bdxio.types";
+
+useHead({ title: `Speakers | ${ASSOCIATION_NAME}` });
+
+const { data: talksWithSpeakers }: {data: Ref<Talk[]>} = await useAPI("/talks", { params: {
+  "populate": "speakers",
+  "pagination[limit]": 100,
+  "filters[edition][year][$eq]": EDITION,
+  "fields[0]": "speakers",
+} });
+
+const speakers = computed(() => {
+  if (!talksWithSpeakers.value.length) return [];
+
+type SpeakerWithTalkId = Array<Speaker & {
+talkId: number;
+}>;
+
+return talksWithSpeakers.value.reduce((acc: SpeakerWithTalkId, talk: Talk) => {
+  if (!talk.speakers.length) return acc;
+
+  console.log(talk);
+
+  talk.speakers.forEach(speaker => acc.push({ ...speaker, talkId: talk.id }));
+  return acc;
+}, []).sort((a,b) => a.name.localeCompare(b.name));
+});
+</script>
+
+<template>
+  <main class="p-section bg-white flex flex-col">
+    <Heading
+      level="1"
+      class="title relative block mx-auto !mb-16"
+    >
+      Les speakers
+    </Heading>
+    <p v-if="speakers.length === 0">
+      Aucun speaker pour l'édition {{ EDITION }}
+    </p>
+    <ul
+      v-else
+      class="flex flex-row flex-wrap items-start justify-start"
```

Comment:
```suggestion
      class="flex flex-row flex-wrap items-center justify-start"
```

Pour que l'image soit centrée avec le nom, prénom et les RS